resallocserver/manager.py

```diff
@@ -349,7 +349,7 @@ def _list_all_resources(self):
             ltype="list",
             catch_stdout_bytes=512,
         )
-        return result["stdout"].decode("utf-8").strip().split("\n")
+        return result["stdout"].decode("utf-8").strip().split()
```

Comment:
Is there a possibility for resource names to contain space? This would be a problem then.